### ISSUE TITLE
Some icons don't show correctly

### ISSUE THREAD

**Issue by aviRon012** [devblackops/Terminal-Icons#118]
Some icons don't show correctly

## Expected Behaviour
I expected the icons to show correctly.

## Current Behaviour
![Screenshot (6)](https://github.com/devblackops/Terminal-Icons/assets/81820388/c7d4581d-e04f-45f2-8116-e7d2e45d9b29)

## Your Environment

* Module version used: 0.11.0
* Operating System version: windows 11 22H2
* PowerShell version: 7.3.7
* nerd-fonts: CascadiaCode v3.0.2 https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.2/CascadiaCode.zip

**Comment by JFSF** [comment]
I have the same issue:

![image](https://github.com/devblackops/Terminal-Icons/assets/1583154/1c44b14d-5657-414b-9efd-8b1c9b5a15ca)

**Comment by aviRon012** [comment]
I solved this temporarily by reverting back to version 0.10.0

**Comment by Tiberriver256** [comment]
If the icons work on `0.10.0` then you're still on Nerd Font v2. The v3 upgrade was quite problematic for me. I ended up having to uninstall all Nerd Fonts and then reinstall the v3 version before it took.

**Comment by anhtuan219** [comment]
@aviRon012 Which specific font of CascadiaCode in v3.0.2 that you installed? I have exactly this issue (icons of Desktop, Links, Music,... look weird)

I use the latest version of PowerShell (v7.3.8) and Terminal-Icons (v0.11.0)
The problem is my font (it's CascadiaCode too) is not the latest

So I download the latest font (v3.0.2 - version that you use)
And I install the 'CaskaydiaCoveNerdFont-Regular.ttf'
![image](https://github.com/devblackops/Terminal-Icons/assets/78535057/06095769-06a5-4329-a85b-75b3b272e567)

After that, everything looks normal
![image](https://github.com/devblackops/Terminal-Icons/assets/78535057/72ab4076-a1d3-4035-817f-91dd4bb97525)

**Comment by anhtuan219** [comment]
> I have the same issue:
> 
> ![image](https://user-images.githubusercontent.com/1583154/273460850-1c44b14d-5657-414b-9efd-8b1c9b5a15ca.png)

@JFSF You can download the latest version of CascadiaCode and install the 'CaskaydiaCoveNerdFont-Regular.ttf' (like my above answer)

Don't forget to specify that font in the settings for PowerShell :))
![image](https://github.com/devblackops/Terminal-Icons/assets/78535057/2197b7ea-e2cd-4bb6-b650-90bdaa671c59)

**Comment by Tiberriver256** [comment]
For me I had to uninstall all NerdFonts and then reinstall a v3 version.

**Comment by grassleaf99** [comment]
Yep, uninstall the old version first, then install latest version

**Comment by JFSF** [comment]
Yep, after replace all Nerd Fonts with version 3.0 it seems to solved the problem.
